app/javascript/components/timeline-options/timeline-page.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import TimelineOptions from './timeline-options';
+import TimelineChart from './timeline-chart';
+import TimelineTable from './timeline-table';
+import NoRecordsFound from '../no-records-found';
+import { buildUrl, buildChartDataObject, buildDataTableObject } from './timeline-helper';
+
+const TimelinePage = ({ id }) => {
+  const [{
+    timelineFormChoices, timelineResources, timelineChartData, timelineTableData,
```

Comment:
`timelineFormChoices, timelineResources` are used to set the state values but they are not being used. so maybe we can remove them?